src/kbc_modules/cc_kbc/attester/snp/mod.rs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2022 IBM
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use super::Attester;
+use anyhow::*;
+use serde::{Deserialize, Serialize};
+use sev::firmware::guest::types::{AttestationReport, SnpReportReq};
+use sev::firmware::guest::Firmware;
+use sev::firmware::host::types::CertTable;
```

Comment:
With the suggested changes you provided for `virtee/sev` in version `1.1.0`, this will be replaced with a `Vec<CertTableEntry>`